README.rst

```diff
@@ -5,12 +5,12 @@ Sliced imaging data
 Background
 ==========
 
-If we want to store imaging data on the cloud and allow scientists to experiment with this data with interactive local tools (e.g., Jupyter notebooks), we should provide an easy interface to retrieve this data.  We should future-proof this model for extremely large images with multiple dimensions, where users may want to pull slices of this data without having to download the entire image.
+If we want to store imaging data on the cloud and allow scientists to experiment with this data with interactive local tools (e.g., Jupyter notebooks), we should provide an easy interface to retrieve this data.  We should future-proof this model for extremely large image sets with multiple dimensions, where users may want to pull slices of this data without having to download the entire image set.
```

Comment:
Looking back at #11 the nouns we landed on are: collection (e.g., hybridization.json), tilesets(e.g., hybridization-fov_001.json), and tiles. What is an image set? By this do you mean tileset? What is an image partition? By this do you mean the format for a tile?

Reply:
oh i see -- by image sets, you mean i have a sample (some volume, like a brain) and i'm looking at multiple FOVs of this sample. these FOVs are comprised of tensors (high dimensional image stacks). as such, an 'image set' is all of the tensors for all of the FOVs that are used to grok the the sample. is that right? if so i like image sets.

Another pass at this could be something like "... future-proof this model for large sample volumes that are being imaged with multiple high dimensional measurements. We anticipate that users will want to access slices of this data (within and across measurements) without having to download all possible images" 

But if you do this, you lose the word image set :(